masonry/src/event.rs

```diff
@@ -30,14 +30,34 @@ pub enum WindowEvent {
     RebuildAccessTree,
 }
 
+/// An indicator of which pointer button was pressed.
+#[derive(PartialEq, Eq, Clone, Copy, Debug, Hash)]
+#[repr(u8)]
+pub enum PointerButton {
+    /// No mouse button.
+    None,
+    /// Primary button, commonly the left mouse button, touch contact, pen contact.
+    Primary,
+    /// Secondary button, commonly the right mouse button, pen barrel button.
+    Secondary,
+    /// Auxiliary button, commonly the middle mouse button.
+    Auxiliary,
+    /// X1 (back) Mouse.
+    X1,
+    /// X2 (forward) Mouse.
+    X2,
```

Comment:
Not a huge fan of these names, but this is what the web spec uses, so 🤷

Reply:
Same